docker-compose.yml

```diff
@@ -242,14 +245,17 @@ services:
         tag: "desec/rabbitmq"
     restart: unless-stopped
 
-  celery:
+  celery-email:
     build: api
     image: desec/dedyn-api:latest
     init: true
-    command: celery -A api worker -l info --uid nobody --gid nogroup
+    command: celery worker -A api -E -Q email_slow_lane,email_fast_lane,email_immediate_lane -c 8 -n email -l info --uid nobody --gid nogroup
```

Comment:
`-Q`: wildcard possible? (`email_*`)
`-c`: Why concurrency?
`-n`: Sets the hostname. Why not call it `celery_email` or something?

Reply:
In Celery 5, [global options can no longer be positioned after the sub-command](https://docs.celeryproject.org/en/latest/whatsnew-5.0.html#step-1-adjust-your-command-line-invocation), so `-A api` must remain in front of `worker`.

Reply:
No wildcard possible. Concurrency is enabled to account for the i/o bound task of sending emails to use resources efficiently. Note sure where the "hostname" shows up, but it's not the hostname in the unix sense. I haven't seen it outside of Celery context at all, hence I opted for keeping this line as short as possible.